### ISSUE TITLE
Display Ecliptic of Date Grid even when not looking from earth.

### ISSUE THREAD

**Issue by Dave1235** [Stellarium/stellarium#1518]
When you are not observing from Earth, the Ecliptic of Date Grid does not appear. The other grids do, including the Ecliptic of J2000. However it would be useful if it did. Just because your not looking from earth sometimes you want to know in which direction the earth is titled at that date. 
If this is a but I’m using...Version 0.20.3 64 bit, Windows 10
Thanks for volunteering you time.
      Dave

**Comment by alex-w** [comment]
Ecliptic (of date) grid shouldn't be visible outside the Earth by definition. Why you expected to see this grid on other celestial bodies?

**Comment by Atque** [comment]
@alex-w An ecliptic of the planet would be shown by drawing a line through the ASP and the apices of the planet.

**Comment by gzotti** [comment]
I think we have discussed this elsewhere. We did not have other planets "ecliptics" for a decade because they are not defined in the literature. We even have added correct orientation of the planet axes (similar to precession for the earth) only a month ago. If you can work out the algorithm and define a stable method to devise the Spring Equinox point (where presumably we should have our Zero point), we may add it, but for now I have other priorities.

**Comment by Atque** [comment]
@gzotti Maybe the zero point is not needed. For example, the invariable plane has no zero point and no partitions.

**Comment by alex-w** [comment]
> @gzotti Maybe the zero point is not needed. For example, the invariable plane has no zero point and no partitions.

any grid required zero point

**Comment by Atque** [comment]
Ok, that makes it more complicated.

**Comment by axd1967** [comment]
In a first step, a line could be drawn. Without reference points there indeed isn't much grid to draw.

But... the (Earth's) ecliptic is the same everywhere in the solar system (even in the entire universe...), as it is a projection onto infinity. The Sun will still be found on that ecliptic line, but Earth (as will most planets) will occasionally wander off that line.

**Comment by gzotti** [comment]
Wrong. The sun will be on the planet's ecliptic line (indeed, as discussed elsewhere, the plane of which may be defineable from heliocentric position and apex (motion vector), but with the zero point depending on the planet's axis, the math of which I will not look at before summer), but not on earth's ecliptic of date when you display earth's ecliptic grid of date on another planet. (Just as the planet is generally not on earth's ecliptic as seen from earth.) I therefore see no point displaying it. The J2000 ecliptic system however is a reference system that appears potentially useful.

**Comment by Dave1235** [comment]
I came across its absence when I was looking to observer the earth's position in its orbit from the sun. As you know, the earth's position relative to its ecliptic of date determines the moments of the equinoxes and solstices and I was looking to observe these points in time, and others directly. As it happens these moments can be observed by noting the position of the sun from the earth instead. If observing the ecliptic from the sun involves any serious programming I would say it's not worth the time and effort. I thought it would just be a case of setting a variable to True or something. 

 Again, thanks for your work, the program is great.   
         Dave
    On Saturday, February 20, 2021, 9:40:13 AM EST, Georg Zotti <notifications@github.com> wrote:  
 
 


Wrong. The sun will be on the planet's ecliptic line (indeed, as discussed elsewhere, the plane of which may be defineable from heliocentric position and apex (motion vector), but with the zero point depending on the planet's axis, the math of which I will not look at before summer), but not on earth's ecliptic of date when you display earth's ecliptic grid of date on another planet. (Just as the planet is generally not on earth's ecliptic as seen from earth.) I therefore see no point displaying it. The J2000 ecliptic system however is a reference system that appears potentially useful.

**Comment by axd1967** [comment]
A poor man's workaround is to enable orbit lines and trails.

![image](https://user-images.githubusercontent.com/3529789/130112667-d3be45fa-0158-420f-8855-04e3d584d907.png)

**Comment by Dave1235** [comment]
Thanks Alex. Just after reading your email, while checking something, I noticed that while the "of date" ecliptic and "of date" ecliptic line do not display while viewing from the sun, the "of j2000" grid and line however do. While it will involve an extra step for me to do what I want to do, it will however get the job done.
 BTW, if you're contributing to the project, thanks for your time.
Dave    On Thursday, August 19, 2021, 01:07:54 PM EDT, alex ***@***.***> wrote:  
 
 


A poor man's workaround is to enable orbit lines and trails.



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Atque** [comment]
> A poor man's workaround to get at least the ecliptic is to enable orbit lines and trails.

A better way is to use the AstroCalc ephemeris generator, and selecting the Sun, and having the time set to the orbital time of the planet in question. Make sure you select "line", and not to take too fine steps.
